src/app/core/services/http-params-mapper.service.ts

```diff
@@ -76,8 +76,13 @@ export class HttpRequestParamsMapper {
       [DataAddressProperty.queryParams]: httpRequestParams.queryParams,
       ...mapKeys(
         httpRequestParams.headers,
-        (k) => `${DataAddressProperty.header}:${k}`,
-      ),
+        (k) => {
+          if(k.toLowerCase() === 'content-type') {
```

Comment:
unformatted, please run npm run format-all

Reply:
Done in https://github.com/sovity/edc-ui/pull/661/commits/672bae1c49d110ac5b996cd4254a81c52ded5203.